addons/mail/static/tests/client_action_tests.js

```diff
@@ -0,0 +1,92 @@
+odoo.define('mail.client_action_test', function (require) {
+"use strict";
+
+var ActionManager = require('web.ActionManager');
+var core = require('web.core');
+var config = require('web.config');
+var testUtils = require('web.test_utils');
+var Widget = require('web.Widget');
+
+var ChatAction = core.action_registry.get('mail.chat.instant_messaging');
+
+QUnit.module('mail', {}, function () {
+
+QUnit.module('ChatAction', {
+    beforeEach: function () {
+        this.data = {
+            "mail.message": {
+                fields: {
+                    id: {type: 'integer', string: 'ID'}
+                }
+            }
+        };
+    }
+});
+
+//--------------------------------------------------------------------------
+// Mobile Test case
+//--------------------------------------------------------------------------
+
+QUnit.test('mobile basic rendering', function (assert) {
+    assert.expect(11);
+
+    // Chaning mobile state
+    config.isMobile = true;
+
+    function createParent (params) {
+        var actionManager = new ActionManager();
+        testUtils.addMockEnvironment(actionManager, params);
+        return actionManager;
+    }
+
+    var action = {
+        id: 1,
+        context: {'active_channel_id': 'channel_inbox', 'active_ids': ['channel_inbox']},
+        params: {'default_active_id': 'channel_inbox'}
+    };
+
+    var parent = createParent({
+        data: {},
+    });
+
+    var chatAction = new ChatAction(parent, action, {});
+    chatAction.set_cp_bus(new Widget());
+
+    testUtils.addMockEnvironment(chatAction, {
+        data: this.data,
+        archs: {
+            'mail.message,false,search': '<search/>',
+        }
```

Comment:
you can add here config: {isMobile: true} (it is one of the argument of addMockEnvironment. and I just noticed it is not documented...)